docs/installation-and-update.md

```diff
@@ -3,5 +3,6 @@
 This section provides instructions on how to:
 
 - [install](installing.md) Percona Distribution for PostgreSQL 14;
+- [migrate](migration.md) from PostgreSQL Community to Percona Distribution for PostgreSQL;
 - [update](minor-upgrade.md) Percona Distribution for PostgreSQL to the latest minor version;
 - [upgrade](major-upgrade.md) Percona Distribution for PostgreSQL to a new major version.
```

Comment:
Minor nit, but don't we start bulleted lists with a capital letter and end them with a dot?

Reply:
Fixed